recipes/graylog-sidecar-repository/recipe.rb

```diff
@@ -0,0 +1,78 @@
+require_relative '../tools'
+
+class GraylogRepository < FPM::Cookery::Recipe
```

Comment:
Can you rename this to `GraylogSidecarRepository`?
```suggestion
class GraylogSidecarRepository < FPM::Cookery::Recipe
```